lib/runner/ConfigurationKeys.js

```diff
@@ -3,6 +3,14 @@ const ConfigurationKeys = [
         key: "config",
         text: "set the configuration path",
     },
+    {
+        key: "projectId",
+        text: "set the projectId, represent your voice app",
+    },
+    {
+        key: "runId",
```

Comment:
the runId should not be set by the user. What I meant when I said to add it to the configuration file, is to set it up as part of the Configuration object generated on Configuration.js it will still be a uuid generated when the tests are run, but it won't be generated in TestRunner.js and so for a single test with multiple files it will only be generated once.